### ISSUE TITLE
Drop the navbar

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-sphinx-theme#89]
<img width="1015" alt="Screenshot 2024-05-28 at 7 59 58 AM" src="https://github.com/dask/dask-sphinx-theme/assets/306380/7f3b035e-9a24-45de-adac-02b0ab891c5e">

The current navbar has links to pages that I think mostly don't matter.  

-  distributed docs are way too technical for most users
- dask-ml is not actually all that focused on dask + ml users do today
- examples is ok, but doesn't get great engagement
- ecosystem a page with a bunch of links on it
- community is intended well, but doesn't seem to actually get people to do community things

The only link I actually think is highly valuable there is the "Dask" link, which will bring people back to the main docs if they're on a subproject site (like dask-jobqueue).  That could probably be managed through an image/link on the top of the  left-sidebar.  

I'm also open to new links if people have ideas.  If we can't find a good list though then I think it's good to focus peoples' attention on the TOC.

cc @scharlottej13 @jacobtomlinson @jrbourbeau

**Comment by mrocklin** [comment]
Trying to be constructive, some core pages:

1.  dask 
2. dataframes
3. arrays
4. deployment
5. Social icons (youtube, social, slack, team (which goes to the community page))

**Comment by scharlottej13** [comment]
Yeah, I definitely don't use the links in the top nav to get to places I want to go. Maybe examples, but as you've mentioned it doesn't get great engagement. From looking at hotjar it seems like people are also more likely to rely on the left TOC too (this is for https://docs.dask.org/en/stable/)

<img width="789" alt="Screenshot 2024-05-30 at 6 09 43 PM" src="https://github.com/dask/dask-sphinx-theme/assets/8620816/2fe3c3ff-c38d-464d-8be0-741cf057cea2">

> Trying to be constructive, some core pages:
> 
> 1. dask
> 2. dataframes
> 3. arrays
> 4. deployment
> 5. Social icons (youtube, social, slack, team (which goes to the community page))

and github? 

I was also looking at other docs for inspiration and many of them (eg polars, prefect, xarray, rapids) all have a pretty minimal top nav and seem to primarily rely on the left TOC.
